src/components/Form.js

```diff
@@ -1,20 +1,76 @@
-import React from 'react';
-
-const Form = () => (
-  <section className="form-container">
-    <h2>ADD NEW BOOK</h2>
-    <div className="form-inputs">
-      <input type="text" placeholder="Book title" required />
-      <select name="categories" id="categories">
-        <option value="Category 1">Novel</option>
-        <option value="Category 2">Essay</option>
-        <option value="Category 3">History</option>
-        <option value="Category 2">Biography</option>
-        <option value="Category 3">LItterature</option>
-      </select>
-      <button type="button">ADD BOOK</button>
-    </div>
-  </section>
-);
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const Form = () => {
+  const initialFormState = {
+    title: '',
+    author: '',
+    category: '',
+  };
+
+  const [formState, setFormState] = useState(initialFormState);
+
+  const handleChange = (e) => {
+    setFormState({
+      ...formState,
+      [e.target.name]: [e.target.value],
+    });
+  };
+
+  const dispatch = useDispatch();
+
+  const submitBookToStore = (event) => {
+    event.preventDefault();
+    document.querySelector('form').reset();
+
+    const newBook = {
+      id: uuidv4(),
+      title: formState.title[0],
+      author: formState.author[0],
+      category: formState.author[0],
```

Comment:
- [ ] I think you meant to have `category: formState.category[0],` here. Kindly fix that.

Reply:
@TedLivist thank you very much. Thanks to your review I found the bug that prevented me to display the category of the book.